### PR TITLE
Thread pool config dto test

### PR DESCRIPTION
**Pull request by asdasd-mjeesh**

### CODE REVIEW

File: src/main/java/executor/service/model/ThreadPoolConfigDto.java [hunk 1]
```diff
@@ -3,24 +3,11 @@
...
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(corePoolSize, keepAliveTime);
-    }
-
-    public ThreadPoolConfigDto(Integer corePoolSize, Long keepAliveTime) {
+    public ThreadPoolConfigDto(Integer corePoolSize, Long
+            keepAliveTime) {
```
Comment (maksymivanov2510) [inline src/main/java/executor/service/model/ThreadPoolConfigDto.java:10]: Let's keep a constructor declaration on the same line (there are not many parameters to split it)

Comment (maksymivanov2510) [inline src/main/java/executor/service/model/ThreadPoolConfigDto.java:10]: Let's add a constructor by default

File: src/test/java/executor/service/model/ThreadPoolConfigDtoTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public void sameHashCodeTest() {
+        assertEquals(threadPoolConfig.hashCode(), secondThreadPoolConfig.hashCode());
+    }
+
+    @Test
+    public void differentHashCodeTest() {
+        secondThreadPoolConfig.setKeepAliveTime(1338L);
+        assertNotEquals(threadPoolConfig.hashCode(), secondThreadPoolConfig.hashCode());
+    }
+}
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ThreadPoolConfigDtoTest.java:41]: Let's add 2 tests for empty object to check 'equals' and 'hashCode' methods.
Use default constructor to do these checks

Comment (asdasd-mjeesh) [reply src/test/java/executor/service/model/ThreadPoolConfigDtoTest.java:55]: > Let's add 2 tests for empty object to check 'equals' and 'hashCode' methods. Use default constructor to do these checks

Done

Comment (asdasd-mjeesh) [reply src/test/java/executor/service/model/ThreadPoolConfigDtoTest.java:55]: > Let's add 2 tests for empty object to check 'equals' and 'hashCode' methods. Use default constructor to do these checks
Done

**Review by maksymivanov2510 — APPROVED** [review]
Looks good! Let's merge the branch
